### DISCUSSION TITLE
External Data Field in the "Bulk Edit family" tab

### DISCUSSION THREAD

**Discussion by anjalijain22** [broadinstitute/seqr#4538] [Q&A] [answered]
Hi,

I have a quick question: I see the field "External Data" for bulk editing family info, however when I try to add a value to it, it gives me an error message. What could be the likely reason for this? Is seqr expecting a certain keyword in that field?

<img width="1051" alt="Screenshot 2024-12-11 at 4 55 06 PM" src="https://github.com/user-attachments/assets/f76d9d02-49c6-4f64-917a-f0bab348ab42" />

Many thanks!

**Comment by anjalijain22** [comment]
Hi, I quickly wanted to follow up on my above question. Appreciate your help on this!

**Comment by hanars** [comment, marked as answer]
This field should not be used by local installations, it is for internal Broad tracking only. I have created https://github.com/broadinstitute/seqr/issues/4550 to track converting this to an internal-only field to prevent future confusion

In general, just because a field is available for use in seqr does not mean it will be meaningful or useful to your team's use case. Adding lots of metadata to a project can be a lot of work for your team to do and maintain, so unless you are sure that adding structured data will be meaningful and useful for your analysts I would strongly recommend you leave out optional columns. For example, MONDO ID is being left as an option for local installs to edit because there are certain seqr installations that are part of a consortium with strict reporting requirements and being able to store this data helps them generate those reports. However, if you do not need those reports you should not be using that column at all - if you do have information about the disease associated with a specific family you should include that in human-readable form in the family description or in case notes

**Comment by hanars** [comment]
This has been removed from teh UI in the latest seqr release
